Tests/Test-COperatingSystem.Tests.ps1

```diff
@@ -0,0 +1,107 @@
+
+#Requires -Version 5.1
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+if( -not (Test-Path -Path 'variable:IsWindows') )
+{
+    $IsWindows = $true
+    $IsLinux = $false
+    $IsMacOS = $false

```

Comment:
Maybe one or two tests without setting these, to ensure it works when they're missing?